### ISSUE TITLE
Docs: progress bar and progress spinner examples are unreachable with NVDA and Firefox

### ISSUE THREAD

**Issue by mfairchild365** [angular/components#15017]
#### What is the expected behavior?
Someone who is using the NVDA screen reader in conjunction with Firefox should have access to the information provided by the progress bar and progress spinner components.

#### What is the current behavior?
Someone who is using the NVDA screen reader in conjunction with Firefox will not be able to access the information provided by the progress bar and process spinner components.

##### For Progress bar
1. Using NVDA 2018.4.1 + Firefox 64
2. Open the [progress bar example](https://material.angular.io/components/progress-bar/examples)
3. Press the 'h' key until you reach the 'results' heading
4. Press the down arrow key to read the next item
4. Observe that the progress bar is not announced

##### For Progress spinner
1. Using NVDA 2018.4.1 + Firefox 64
2. Open the [progress bar example](https://material.angular.io/components/progress-spinner/examples)
3. Press the 'h' key until you reach the 'results' heading
4. Press the down arrow key to read the next item
4. Observe that the progress bar is not announced

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
- Angular Material Example 7.2.2 located at https://material.angular.io/components/
- NVDA 2018.4.1 + FireFox 64
- VoiceOver for IOS (12.1.3) + Safari (12.1.3)

#### Is there anything else we should know?
The progress bar and spinner can be accessed in other screen readers, including VoiceOver for iOS.
